MapleServer2/PacketHandlers/Game/StatPointHandler.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.Extensions.Logging;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Servers.Game;
+using MapleServer2.Packets;
+using System;
+
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class StatPointHandler : GamePacketHandler
+    {
+        public override RecvOp OpCode => RecvOp.STAT_POINT;
+
+        public StatPointHandler(ILogger<StatPointHandler> logger) : base(logger) { }
+
+        public override void Handle(GameSession session, PacketReader packet)
+        {
+            byte mode = packet.ReadByte();
+
+            switch (mode)
+            {
+                case 2: 
+                // Increment attribute point
```

Comment:
either put it same line as case 2 or indent it to go with the next line. probably the former is better

Reply:
Indented and fixed.